src/types/amenity.types.ts

```diff
@@ -0,0 +1,5 @@
+export interface AmenityInterface {
```

Comment:
if your pr is merged before Begzod's, don't forget to tell him to remove this type because he has identical type in his PR. Or if his PR is merged before yours, delete this type and just use the type that Begzod created.

Reply:
they should be identical actually yes

Reply:
why string | Date and not just Date ?

Reply:
just string is better I think because their type will be string after getting response body data from backend. axios does not parse json date string to js date object